css/site.styl

```diff
@@ -322,6 +322,23 @@ body.menu-active
   cursor: pointer
   
 // ******************************************************
+
+// SOBRE NOSOTROS
+
+.prensa-files
+  li,
+  a
+    display: inline-block
+
+  a
+    font-size: $font-size-larger
```

Comment:
you could make a class for the `a` like `.prensa-file` instead of nesting it under the `ul` in css